### DISCUSSION TITLE
Running Fan Control suddenly makes my PC lag.

### DISCUSSION THREAD

**Discussion by PlatyPuzz** [Rem0o/FanControl.Releases#2393] [Q&A]
So this just happened when I turned on my PC. I have never had this issue before, but now suddenly after having my computer off for a few hours, when I turned it back on Fan Control makes it lag. I turned off Fan Control which makes everything normal again. I cant find anything on the web for this. Anyone have any idea?

**Comment by Rem0o** [comment]
Disable the "controller" section in your sensor sources.

**Reply by PlatyPuzz** [reply]
Has always been disabled. Tried enabling and disabling again, same issue.

**Reply by Rem0o** [reply]
Storage? Basically narrow down which source causes lag.

**Comment by PlatyPuzz** [comment]
I did a reinstall. And now Fan control only detects my video card and nothing else. I have no clue as to what as happened...
    På fredag 22. mars 2024 kl. 14:09:15 CET skrev Rem0o ***@***.***> følgende:  
 
 


Storage? Basically narrow down which source causes lag.

**Comment by PlatyPuzz** [comment]
I dont know what's going on. But now suddenly my fans appear again, with the next "no speed sensor" and I tried checking and unchecking every source. The source is the motherboard. Do you have any idea?
    På fredag 22. mars 2024 kl. 23:15:46 CET skrev Yahoo ***@***.***> følgende:  
 
  I did a reinstall. And now Fan control only detects my video card and nothing else. I have no clue as to what as happened...
    På fredag 22. mars 2024 kl. 14:09:15 CET skrev Rem0o ***@***.***> følgende:  
 
 


Storage? Basically narrow down which source causes lag.

**Comment by PlatyPuzz** [comment]
So i reconnected my fans. Opened my PC, took them out of their 4 pins and then connected them again. Now everything seems to work like it used to. No clue as to what really made this happen, but this seems to have fixed it.
